### PR TITLE
added PHP section

### PR DESCRIPTION
**Pull request by robcmo**
* added PHP
* MarkDown code cleanup
* fixes trailing spaces

### CODE REVIEW

**Comment by robcmo** [comment]
Please help with error `MD012 Multiple consecutive blank lines`.
This does not appear to occur in Notepad++ in online GitHub editor/viewer.

**Comment by netniV** [comment]
Basically, there should never be two blank lines, one after the other.  If there is, the additional lines should be removed. I don't think that applies to code blocks.  Normally the failing check will show the line in question in this case 134.

File: Upgrading-Cacti-Under-Windows.md [hunk 1]
```diff
@@ -22,13 +25,24 @@ Download latest cacti-xxx.zip
 
 4. Windows Update
 
-5. MySQL Installer - update Catalog, then upgrade MySQL Server
+5. MySQL
+
+   * Open `MySQL Installer - Community`, update Catalog,
```
Comment (netniV) [inline Upgrading-Cacti-Under-Windows.md]: Since we have separated out the steps, may as well fully separate them using bullet points.

**Comment by netniV** [comment]
If you pull the latest changes that have been made, the double line issue should be resolved now as it was in a file you were not modifying.

**Comment by robcmo** [comment]
I had corrected the issue on 3/28, but the automated testing did not re-run. Did I miss something or do I have to close and start a new pull request?

**Comment by netniV** [comment]
Did you re push to your repo branch?

**Pull request merged**